MotionMark/tests/core/resources/suits.js

```diff
@@ -97,15 +96,15 @@ window.SuitsParticle = Utilities.createSubclass(Particle,
     }
 });
 
-var SuitsStage = Utilities.createSubclass(ParticlesStage,
-    function()
+class SuitsStage extends ParticlesStage {
+    constructor()
     {
-        ParticlesStage.call(this);
-    }, {
+        super();
+    }
 
-    initialize: function(benchmark)
+    initialize(benchmark)
     {
-        ParticlesStage.prototype.initialize.call(this, benchmark);
+        super.initialize(this, benchmark);
```

Comment:
The `Stage.initialize()` takes only `benchmark` as an argument. Passing this to `super.initialize()` seems to be wrong.

Reply:
Passing `this` here is a class conversion mistake. Will fix.

Reply:
Fixed in a later commit in this PR.